whois-commons/src/main/java/net/ripe/db/whois/common/apiKey/ApiKeyUtils.java

```diff
@@ -35,7 +37,11 @@ public static boolean validateScope(final OAuthSession oAuthSession, final List<
 
         final OAuthSession.ScopeFormatter scopeFormatter = new OAuthSession.ScopeFormatter(whoisScope.get());
 
-        if(StringUtils.isEmpty(scopeFormatter.getScopeKey()) || StringUtils.isEmpty(scopeFormatter.getScopeType()) || StringUtils.isEmpty(scopeFormatter.getAppName())) {
+        if (!StringUtil.isNullOrEmpty(scopeFormatter.getScopeEnv()) && !environment.equalsIgnoreCase(scopeFormatter.getScopeEnv())){
```

Comment:
No need to check for null or empty

Reply:
This doesn't apply now, the code has changed